default.css

```diff
@@ -136,6 +138,20 @@ h6 {
   font-size: 11.12rem;
 }
 
+
+input,
+textarea {
+  border: 2px solid var(--black);
+  padding: 6px;
+  border-radius: 5px;
+}
+
+input[type="checkbox"],
+input[type="radio"] {
+  cursor: pointer;
+  accent-color: var(--black);
+}
```

Comment:
I think here you can give your input some default class and then style it. otherwise if someone wants to use your inputs and does not want your default black checkbox and radio. This styles will be add by default. user will get confuse were are the styles coming and all. 
I may suggest create a separate class for input and style it.

Reply:
Hi Miheer, thanks for reviewing. I will remove the accent color.